join_array.rb

```diff
@@ -0,0 +1,11 @@
+# joining array without using inbuilt function
+
+arr1 = Array(0..5)
+arr2 =  Array(6..10)
+result = arr1 + arr2
+puts "#{result}"
```

Comment:
No need to use " " and { } if you wish to print only array. This is used when array is used in between the string.